mask/README.md

```diff
@@ -0,0 +1,22 @@
+## Introduction
+
+In **CloudMaskTask** example we show how to get cloud masks provided by the Sentinel Hub service
```

Comment:
I'd rewrite to:

In **CloudMaskTask** example we show how to obtain Sentinel-2 cloud masks with eo-learn. 

The simplest (and fastest) way is requesting cloud masks (and optionally cloud probabilities) from the Sentinel Hub service. In case provided cloud masks are not sufficient we also show how to use [`s2cloudless`](https://github.com/sentinel-hub/sentinel2-cloud-detector) to  calculate your own.